src/pages/components/menu/usage.mdx

```diff
@@ -0,0 +1,572 @@
+
+![Menu alignment right to left](images/usage-menu-RTL.png)
+
+</Column>
+</Row>
+
+## Content
+
+### Main elements
+
+#### Overflow
+
+Consider using short and precise labels for menu items. However, when deciding
+truncating long menu items, use a tooltip on hover to reveal the full text.
+
+<Row>
+<Column colLg={8}>
+
+![An overflow example for menu items](images/usage-menu-overflow-content.png)
+
+<Caption>Tooltip helps clarify the truncated text</Caption>
+</Column>
+</Row>
+
+#### Ordering and grouping
+
+To aid discoverability, arrange menu items in a meaningful sequence that aligns
+with the order of operations. To help users focus on the most relevant options,
+place the ones used most often at the top.
+
+When there is a significant amount of menu items, it's advisable to group them
+into sections, separated by divider lines.
+
+<Row>
+<Column colLg={8}>
+
+![Ordering and grouping](images/usage-menu-ordering.png)
+
+<Caption>
+  Example shows how menu items are grouped into meaningful sections.
+</Caption>
+</Column>
+</Row>
+
+#### Number of items within a menu
+
+Expansive lists of options in menus can overwhelm users. With context menu and
+overflow menu, keep the list length manageable by including no more than 12
+items to help users scan through items easily without having to scroll. With
+menus used under [menu buttons], keep the list smaller with under 5 action
+items.
+
+<Row>
+<Column colLg={4}>
+
+![Do consider including as fewer items as possible](images/usage-menu-number-items-do.png)
+
+<Caption>
+  Do consider including as few items as possible
+</Caption>
+</Column>
+
+<Column colLg={4}>
+
+![Do not overload choices](images/usage-menu-number-items-dont.png)
+
+<Caption>
+ Do not overload choices
+</Caption>
+</Column>
+</Row>
+
+### Submenus
+
+A submenu is a menu item with the caret icon on the right, offering additional
+related commands. While submenus can help tidy up a context menu and make its
+options clearer, introducing multiple levels of submenus can make the user
+experience more complex and harder to navigate. Avoid multiple levels of nesting
+when it comes to submenus.
+
+#### Actions associated with a shared label
+
+When a term appears in more than two menu items in the same group, consider
+eliminating the recurring term in the submenu label to help people predict its
+content and scan easily.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do omit recurring term">
+
+![Do omit recurring term](images/usage-image-submenus-title-do.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not repeat the same term from the submenu trigger.">
+
+![Do not repeat the same term from the submenu trigger](images/usage-image-submenus-title-dont.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Further guidance
+
+For further content guidance, see Carbon's
+[content guidelines](/guidelines/content/overview).
+
+## Universal behaviors
+
+### States
+
+Menu items have seven main progress states– enabled, hover, focus, focus and
```

Comment:
> Menu items have seven main progress states–

I would probably use a colon here instead of a dash, like you did earlier on this page for Sizes.

Reply:
Did you want to semi-bold these states?